
def test(num):
    if num==2:
        raise ValueError('I am coming from test 2')

test(2)